apps/charterafrica/src/payload/blocks/Documents.js

```diff
@@ -0,0 +1,254 @@
+              label: {
+                en: "Organization",
+                fr: "Organisation",
+                pt: "Organização",
+              },
+            }),
+          ],
+        },
+        {
+          type: "collapsible",
+          label: {
+            en: "Filter Bar",
+            fr: "Barre de filtre",
+            pt: "Barra de filtro",
+          },
+          fields: [
+            {
+              name: "filterBar",
+              type: "group",
+              admin: {
+                hideGutter: true,
+              },
+              fields: [
+                {
+                  name: "search",
+                  type: "group",
+                  admin: {
+                    hideGutter: true,
+                  },
+                  fields: [
+                    {
+                      name: "label",
+                      type: "text",
+                      label: {
+                        en: "Search Label",
+                        fr: "Étiquette de recherche",
+                        pt: "Rótulo de pesquisa",
+                      },
+                      defaultValue: defaultValue({
+                        en: "Search",
+                        fr: "Rechercher",
+                        pt: "Pesquisar",
+                      }),
+                      required: true,
+                      localized: true,
+                    },
+                  ],
+                },
+                {
+                  name: "sort",
+                  type: "group",
+                  admin: {
+                    hideGutter: true,
+                    style: {
+                      marginTop: 10,
+                    },
+                  },
+                  fields: [
+                    {
+                      name: "label",
+                      type: "text",
+                      label: {
+                        en: "Sort Label",
+                        fr: "Étiquette de tri",
+                        pt: "Rótulo de classificação",
+                      },
+                      defaultValue: defaultValue({
+                        en: "Sort",
+                        fr: "Trier",
+                        pt: "Classificar",
+                      }),
+                      required: true,
+                      localized: true,
+                    },
+                    {
+                      name: "options",
+                      type: "array",
+                      minRows: 1,
+                      label: {
+                        en: "Sort Options",
+                        fr: "Options de tri",
+                        pt: "Opções de classificação",
+                      },
+                      fields: [
+                        {
+                          name: "value",
+                          type: "select",
+                          unique: true,
+                          required: true,
+                          options: documentSortOptions.map((value) => {
```

Comment:
`options`  Can either be an array of strings, or an array of objects containing a label string and a value string.
So you could as well have this as 
```suggestion
                          options: documentSortOptions,
```